code/lib/cli/src/helpers.ts

```diff
@@ -197,14 +197,15 @@ export async function copyComponents(
   };
   const componentsPath = async () => {
     const baseDir = getRendererDir(renderer);
-    const assetsDir = join(baseDir, 'template/cli');
-    const assetsLanguage = join(assetsDir, languageFolderMapping[language]);
-    const assetsJS = join(assetsDir, languageFolderMapping[SupportedLanguage.JAVASCRIPT]);
-    const assetsTSLegacy = join(
+    const assetsDir = posix.join(baseDir, 'template/cli');
```

Comment:
When do we need to use posix.join instead of path.join?

Reply:
posix.join should be used, if you want to use the path seperator of posix systems, independently, on which system you are. 

In this case though it doesn't make much sense, because all the paths are used to be passed to fse.copy, which can handle path separators on both systems. My first assumption was, that these paths are used to resolve node modules with node's resolution algorithm. Therefore it had made sense to use forward slash, because we don't require packages with a backslash. This assumption was wrong and therefore I have reverted the these changes.